src/App.tsx

```diff
@@ -1,9 +1,81 @@
-import styles from "./App.module.css";
+import FlightTable from "./FlightTable";
+import { faker } from "@faker-js/faker";
+import { useState } from "react";
 
-const App = () => (
-  <div className={styles.mainContainer}>
-    <h1 className={styles.header}>Welcome to our site!</h1>
-  </div>
-);
+export interface Flight {
+  flightNumber: string;
+  departure: string;
+  destination: string;
+  flightDuration: string;
+  departureTime: string;
+  arrivalTime: string;
+}
+
+const App = () => {
+  const [flightList, setFlightList] = useState([] as Flight[]);
+
+  const generateRandomTime = (): string => {
+    const hours: number = faker.datatype.number({ min: 0, max: 23 });
+    const minutes: number = faker.datatype.number({ min: 0, max: 59 });
+    return `${String(hours).padStart(2, "0")}:${String(minutes).padStart(
+      2,
+      "0",
+    )}:00`;
+  };
+
+  const calculateDuration = (start: string, end: string): string => {
+    const [startHours, startMinutes] = start.split(":").map(Number);
+    const [endHours, endMinutes] = end.split(":").map(Number);
```

Comment:
This is very clever, destructuring the array immediately is a great way to save lines of code!